packages/react-dom/src/events/__tests__/SyntheticWheelEvent-test.js

```diff
@@ -83,7 +92,7 @@ describe('SyntheticWheelEvent', () => {
     expect(events[1].deltaY).toBe(-50);
   });
 
-  it('should be able to `preventDefault` and `stopPropagation`', () => {
+  it('should be able to `preventDefault` and `stopPropagation`', async () => {
```

Comment:
Can add an `expect.assertions(3)` to the end of this test

Reply:
Shouldn't the expected number of assertions be 5? 2x for each dispatchEvent call (4 total) and one expect events to be "2".

Reply:
Yeah I counted wrong